index.html

```diff
@@ -209,18 +209,24 @@ <h3 id="matters-considered-by-the-placement-rules">Considerations for  simple pl
     and will use the same logic in either case.
     Specifically, the center of the ruby string and of the base character 
     string are aligned in the inline direction for mono-ruby.</li>
-<li id="l20200529009">Processing is done in two steps. 
-    In the first step, processing of layout only considers  the relative positions of the ruby 
-    annotation and its base text. 
-    In the second step,  layout processing decides the position of the <a title="ruby block">ruby block</a> in the line, taking into consideration the preceding and 
-    following characters. In other words relative positions of the ruby annotation and its base text as decided in the first step are not modified 
-    in  light of any characters preceding and following the ruby block. 
-    Also, when the ruby block is placed at 
-    the line head or the line end the method used in this document does not align the first or last 
-    character of the base text to the line head or the line 
-    end by modifying the relative positions of the ruby annotation and its base text. 
-    To summarise, the relative positions decided in the first step are not 
-    modified by the second step at all.</li>
+<li id="l20200529009">Processing is done in two levels.
+    The first level only determines relative positions of ruby annotation and its base.
+    Meanwhile, the second level determines inline positions of <a>ruby boxes</a>.
+    In other words, relative positions of the ruby annotation and its base text as decided in the first level
```

Comment:
I think we can drop 'Meanwhile'.  Either that or replace it with 'On the other hand' if you want to stress the distinction, though i think having 2 levels already does that.

Reply:
I just read the previous version of this sentence, and it seems much clearer.  What was the goal in rewriting it?